native/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -0,0 +1,147 @@
+            String projectName = balProject.currentPackage().descriptor().name().value();
+            name = orgName + "." + projectName;
+        } catch (ProjectException e) {
+            errStream.println("The current directory is not a Ballerina project!");
+            return;
+        }
+        if (!Files.exists(Paths.get(sourcePath, configPath))) {
+            try {
+                createConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when creating the Config.toml file: " + e.getMessage());
+            }
+        } else {
+            try {
+                updateConfigToml();
+            } catch (Exception e) {
+                errStream.println("Failure when updating the Config.toml file: " + e.getMessage());
+            }
+        }
+    }
+    private void createConfigToml() throws Exception {
+        SyntaxTree syntaxTree = CreateSyntaxTree.createToml();
+        writeOutputFile(syntaxTree, Paths.get(this.sourcePath, "Config.toml").toAbsolutePath().toString());
+    }
+
+    private void updateConfigToml() throws Exception {
+        SyntaxTree syntaxTree = CreateSyntaxTree.updateToml(Paths.get(this.sourcePath, this.configPath), this.name);
+        writeOutputFile(syntaxTree, Paths.get(this.sourcePath, "Config.toml").toAbsolutePath().toString());
+    }
+
+    private void writeOutputFile(SyntaxTree syntaxTree, String outPath) throws Exception {
+        String content = "";
+        Path pathToFile = Paths.get(outPath);
+        Files.createDirectories(pathToFile.getParent());
+        try {
+            content = Formatter.format(syntaxTree.toSourceCode());
+        } catch (FormatterException e) {
+            throw e;
+        }
+        try (PrintWriter writer = new PrintWriter(outPath, StandardCharsets.UTF_8.name())) {
+            writer.println(content);
+        } catch (IOException e) {
+            throw e;
+        }
+    }
+
+    public void setSourcePath(String sourceDir) {
+        this.sourcePath = sourceDir;
+    }
+
+    public void setEnvironmentBuilder(ProjectEnvironmentBuilder projectEnvironmentBuilder) {
+        this.projectEnvironmentBuilder = projectEnvironmentBuilder;
+    }
+
+    @Override
+    public void setParentCmdParser(CommandLine parentCmdParser) {
+        this.parentCmdParser = parentCmdParser;
+    }
+    @Override
+    public String getName() {
+        return PersistToolsConstants.COMPONENT_IDENTIFIER;
+    }
+    
+    @Override
+    public void printLongDesc(StringBuilder out) {
+        out.append("Generate database configurations file inside the Ballerina project").append(System.lineSeparator());
+        out.append(System.lineSeparator());
+    }
+    
+    @Override
+    public void printUsage(StringBuilder stringBuilder) {
+        stringBuilder.append("  ballerina " + PersistToolsConstants.COMPONENT_IDENTIFIER +
+                " init" + System.lineSeparator());
```

Comment:
```suggestion
        stringBuilder.append("  ballerina " + PersistToolsConstants.COMPONENT_IDENTIFIER + " init").
                append(System.lineSeparator());
```

Reply:
Updated in [8078951](https://github.com/ballerina-platform/persist-tools/pull/9/commits/8078951dff409765a286bc5e8f289d15b9b412ee)